source/content/core-updates.md

```diff
@@ -99,7 +99,7 @@ This process can potentially cause loss of data. Be sure you have no custom code
 <Tab title="Drupal 9" id="d9" active={true}>
 
 ```bash{promptUser: user}
-git pull -Xtheirs https://github.com/pantheon-systems/drupal-recommended master
+git pull -Xtheirs https://github.com/pantheon-systems/drupal-composer-managed master
```

Comment:
Did you test this & make sure that we don't need `main:master` now? n.b. drupal-composer-managed has a default branch of `main` and the Pantheon repo still has a default branch of `master`. This might necessitate some changes to some of these git commands.